src/rpcblockchain.cpp

```diff
@@ -16,6 +16,10 @@
 #include <stdint.h>
 
 #include <univalue.h>
+#include "utilmoneystr.h"
+#include "base58.h"
+#include "clientversion.h"
```

Comment:
Last thing that I can see off-hand:

1. `utilmoneystr.h` and `base58.h` are already included (lines 13 & 14). 
2. move the `#include "clientversion.h"` up to be above the include of `<stdint.h>` (not a big deal, just a style thing).